helm/cfgov/templates/tests/smoke-test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "smoke-test-job"
```

Comment:
This needs to use dynamic naming, otherwise there will be a resource name conflict when more than one cfgov release is deployed within the same namespace (ergo, EKS). See other tests for example.